juniper_subscriptions/src/lib.rs

```diff
@@ -19,6 +19,11 @@ use juniper::{
     BoxFuture, ExecutionError, GraphQLError, GraphQLSubscriptionType, GraphQLTypeAsync, Object,
     ScalarValue, SubscriptionConnection, SubscriptionCoordinator, Value, ValuesStream,
 };
+#[cfg(feature = "unstable-ws-subscriptions-transport")]
+/// Utilities for the implementation of subscriptions over websocket
+pub mod ws_util;
+#[cfg(feature = "unstable-ws-subscriptions-transport")]
```

Comment:
Not sure, but it isn't just subscriptions, it is all GraphQL over websockets right?

Reply:
Yeah, it seems to be, since i didnt found no other protocol specification about graphql over websocket other than https://github.com/apollographql/subscriptions-transport-ws this one, that still to be a work in progress, i doesnt know what else could i use to actually base this.